lib/katello/tasks/reindex.rake

```diff
@@ -36,6 +36,9 @@ namespace :katello do
       begin
         yield
       rescue Exception => e
+        if object_class == Katello::Erratum || Katello::Util::Search.pulp_backend_search_classes.include?(object_class)
```

Comment:
All of the converted content types include https://github.com/Katello/katello/blob/master/app/models/katello/erratum.rb#L3 so you could check for that? My worry is, this will be missed, Package Groups for example has been merged now.